internal/uvm/scsi.go

```diff
@@ -323,9 +347,41 @@ func (uvm *UtilityVM) addSCSIActual(ctx context.Context, hostPath, uvmPath, atta
 	}
 
 	if err := uvm.modify(ctx, SCSIModification); err != nil {
-		return nil, fmt.Errorf("uvm::AddSCSI: failed to modify utility VM configuration: %s", err)
+		return fmt.Errorf("uvm::AddSCSI: failed to modify utility VM configuration: %s", err)
```

Comment:
Can we change this error wrapping to just `"failed to modify UVM with new SCSI mount: %s"`?